db/seeds.rb

```diff
@@ -0,0 +1,71 @@
+return if Rails.env.test?
+
+gds = Organisation.create!(
+  name: "Government Digital Service",
+  content_id: Organisation::GDS_ORG_CONTENT_ID,
+  organisation_type: :ministerial_department,
+  slug: "government-digital-service",
+)
+
+User.create!(
+  name: "Test Admin",
+  email: "test.admin@gov.uk",
+  password: "6fe552ca-d406-4c54-b7a6-041ed1ade6cd",
+  role: :superadmin,
+  confirmed_at: Time.zone.now,
+  organisation: gds,
+)
+
+test_organisation = Organisation.create!(
+  name: "Test Organisation",
+  content_id: SecureRandom.uuid,
+  organisation_type: :ministerial_department,
+  slug: "test-organisation",
+)
+
+User.create!(
+  name: "Test User",
+  email: "test.user@gov.uk",
+  password: "6fe552ca-d406-4c54-b7a6-041ed1ade6cd",
+  role: :normal,
+  confirmed_at: Time.zone.now,
+  organisation: test_organisation,
+)
+
+# The following user has 2SV enabled by default. Scan the following QR code with your authenticator app to generate a code to login.
+# █████████████████████████████████████████████████
+# █████████████████████████████████████████████████
+# ████ ▄▄▄▄▄ ██▀█▄▄▄▄██▄ █▀ ██▀▄▀█▀█▄█▀█ ▄▄▄▄▄ ████
+# ████ █   █ █▀ ▀▀▀█ █▀█▀▄ ▀  █ ▀▀▄▄ ▄ █ █   █ ████
+# ████ █▄▄▄█ █▄▄▀█▄█  ▄ █▀▄▄ ▄▄▀ ▀▀ ████ █▄▄▄█ ████
+# ████▄▄▄▄▄▄▄█▄█▄▀ █ ▀ █▄▀ █ ▀▄█▄█▄█ ▀▄█▄▄▄▄▄▄▄████
+# ████▄ ▀█▀▄▄ ▀▄██  ▄▀▄ █ ██▄▀ ▄▀▀█▄▀▀█▀█▀ ▄▀▀▀████
+# ████ ▄██ ▀▄ ▀▄ ▀▀ ▄ ▄ ▀██▀ █▄ ▀  ▄▄█▄▀▄ █▀█  ████
+# ████▄  █▄▀▄ ▄▀██▀▄█▄█▄▀▄██ ▄▀▀  █▀▄▀ ▀▀█▀▄▀  ████
+# ████▀█ ██▄▄▄▀█ █▀█ ▄ ▄▀█ ██▀▀▄ ▀█▄█ ▀█▀   ▀█▀████
+# ████▄▀█  █▄▄▄  ▀█▄▀ ▀▄ █▀▀▄ █▄ ▀▀▄▀ ▀█▀█▀██▀▀████
+# ████▄▄▄ ▄ ▄██▄▄▀█▄ ▄▀▄▄  █  ▀█▀   ▀ ██ ▀█▀▄ ▄████
+# ████ █▄  █▄  ▄▄█▄▀█▄█▀▄██▄█▀▀▄██  ▄█  ▄██ ▄ ▄████
+# ████▄▄█▀ ▄▄█ ▀  ▄█ ▀▀▄▀ ▀▄   █ ▄▄▄▀▄█▀ ▀▄▀ ▀▀████
+# ██████  ▄▄▄█▄ ▄▀█▀▄▀ ▀█▀▀█▀ ███▀▄▀  ▀▀ ▄▀ ███████
+# ████▀▄▀▀▀ ▄ ██▄▀      ▀▀ █▄██ ▄ █▀▄▀▄█ ██ ▀ ▀████
+# ████ ▄  ▄▄▄█ ▄▄▄▀█▄▄▄█▀███▀▀▀ ▀ ▄ ▄  ▀█▄▀ ▄█▀████
+# ████ ▄▀▀▄▄▄▄▄█▄▄▀▄ ▄█▄█▄▄█▄  █ ▀▄█▀ ██▄▄▄▀▄▀▄████
+# ████▄█▄▄▄▄▄█   ▀▀█▀ ▄▄▀▀▄▀▀▀▄▄█ ▄ ▀▄ ▄▄▄ ▀ █ ████
+# ████ ▄▄▄▄▄ █ ▀█ ██▀  █▄▀█▀██▄▄ ▄ ▀▄▄ █▄█ ▀██ ████
+# ████ █   █ ██▄▀█▄▄ █  ▄▄▄▀  ▀   ██▄▄▄ ▄ ▄▄▀▀ ████
+# ████ █▄▄▄█ █▀▀▄ ▄▄█▀█▄▄▄▄█ ▀█▄▀▀▀█▀███ █ █ ▄▀████
+# ████▄▄▄▄▄▄▄█▄█▄▄▄▄██▄▄▄█▄████▄▄██▄█▄▄▄███████████
+# █████████████████████████████████████████████████
+# █████████████████████████████████████████████████
```

Comment:
This QR code doesn't look right on GitHub as they add line spacing.  It's fine if you open the file with less, vim or cat to a terminal.